ngx-fudis/projects/ngx-fudis/src/lib/foundations/motion/tokens.scss

```diff
@@ -0,0 +1,23 @@
+$duration-short1: 50ms;
+$duration-short2: 100ms;
+$duration-short3: 150ms;
+$duration-short4: 200ms;
+$duration-medium1: 250ms;
+$duration-medium2: 300ms;
+$duration-medium3: 350ms;
+$duration-medium4: 400ms;
+$duration-long1: 450ms;
+$duration-long2: 500ms;
+$duration-long3: 550ms;
+$duration-long4: 600ms;
```

Comment:
Nice addition for motion tokens but for now let's not clutter the codebase until these have gone through our designers. Leave just that one token which is in use in icon component.

Change the naming of the token for something more intuitive, e.g _$duration-600_